### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by katyhuff**
@xwa9860 : want to review a pull request? this one is a simple one that addresses two bugs. One is the one noticed by you and manuele in dtemp. the other is an improper import that was leftover cruft from the reorganization I did.

### CODE REVIEW

File: pyrk/th_component.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def dtemp(self):
         if self.prev_t_idx == 0:
             return 0.0*units.kelvin
         else:
-            return (self.T[self.prev_t_idx] - self.T[self.prev_t_idx-1])
+            return (self.T[self.prev_t_idx] - self.T0)
```
Comment (xwa9860) [inline pyrk/th_component.py]: This is good. But it is going to be changed to some steady state temperature when we merge the branch with_units in, which is even better for computation stability.

**Comment by xwa9860** [comment]
Sorry this took so long. I finally got time.

File: pyrk/th_component.py (hunk 1 above)
Comment (katyhuff) [inline pyrk/th_component.py]: yep. I'm happy to wait for with_units . Up to you.

**Comment by katyhuff** [comment]
@xwa9860 , Shall we merge this for now, or wait for #10 ?

**Comment by xwa9860** [comment]
I think we can wait for 10, which is very soon, hopefully.

On Sun, Sep 27, 2015 at 11:16 AM, Katy Huff <notifications@github.com>
wrote:

> @xwa9860 <https://github.com/xwa9860> , Shall we merge this for now, or
> wait for #10 <https://github.com/pyrk/pyrk/pull/10> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyrk/pyrk/pull/17#issuecomment-143582072>.
>



-- 
-----------------------------------------
Xin WANG | 王昕
PhD Student
Thermal Hydraulics Laboratory
Nuclear Engineering Department
University of California, Berkeley
(415)408-1691

**Pull request closed without merging**